src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/MergeAssetManifests.cs

```diff
@@ -33,7 +33,13 @@ public class MergeAssetManifests : Task
         /// </summary>
         public string VmrBuildNumber { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Vmr Vertical Name, e.g. "Android_Shortstack_arm". Allowed to be empty for non official builds.
+        /// </summary>
+        public string VerticalName { get; set; } = string.Empty;
```

Comment:
I don't think it will ever be null. We will always be setting it when calling this task, either to an explicit value, or `string.empty`, since that's the value of unassigned properties. Might be wrong here tho